src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -99,6 +99,20 @@ public MainViewModel getMainViewModel(@Nullable Project project) {
     return mainViewModels.computeIfAbsent(project, this::createNewMainViewModel);
   }
 
+  /**
+   * Triggers a main view model update for the main view model corresponding to the given project.
+   * If the project is null, this method does nothing.
+   */
+  public void updateMainViewModel(@Nullable Project project) {
+    if (project == null) {
+      return;
```

Comment:
Ain't there `MainViewModel` for null project, too? Okay, perhaps it irrelevant, in any case.

Reply:
A main view model yes, but not a main view model updater, since the null project is interpreted as the default project.